webportal/lambda/deployment/cli_parser.py

```diff
@@ -0,0 +1,10 @@
+import argparse
+
+VALID_ENVIRONMENTS = ['dev', 'test', 'prod', 'production']
```

Comment:
It might be suitable to put a comment here _(or wherever you think appropriate)_ explaining why we've got `prod` and `production` as allowable values.